src/main/java/com/movie/movie/entity/Movie.java

```diff
@@ -1,43 +1,42 @@
-package com.movie.movie;
-
-public class Movie {
-    private int id;
-    private String title;
-    private int release_year;
-
-    public Movie(int id, String title, int release_year) {
-        this.id = id;
-        this.title = title;
-        this.release_year = release_year;
-
-
-    }
-
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        this.id = id;
-    }
-
-    public String getTitle() {
-        return title;
-    }
-
-    public void setTitle(String title) {
-        this.title = title;
-    }
-
-    public Integer getRelease_year() {
-        return release_year;
-    }
-
-    public void setRelease_year(int release_year) {
-        this.release_year = release_year;
-    }
-}
-
-
-
-
+package com.movie.movie.entity;
+
+public class Movie {
+    private int id;
+    private String title;
+    private int release_year;
```

Comment:
Javaの変数はキャメルケースでお願いします。
```suggestion
    private int releaseYear;
```

Reply:
データベースにおけるテーブルのカラム名はスネークケースでOK
ややこしい。

Reply:
対応しました。
e7f49a0b4e40c86ee58b8556557c727696e42593